packages/python/plotly/plotly/express/_core.py

```diff
@@ -41,6 +41,30 @@ def __init__(self):
 defaults = PxDefaults()
 del PxDefaults
 
+
+class IdentityMap(object):
```

Comment:
could these two classes be moved to a `_special_inputs.py` file to shorten a bit `_core.py`?

Reply:
sure, if needed

Reply:
as you wish, I just feel that the length of this file makes it a bit overwhelming.

Reply:
done!